cmp/grid/impl/ColumnWidthCalculator.js

```diff
@@ -117,8 +130,20 @@ export class ColumnWidthCalculator {
 
         // 4) Render to a hidden cell to calculate the max displayed width
         return reduce(longestValues, (currMax, value) => {
-            const width = this.getCellWidth(value, renderer) + indentationPx + bufferPx;
-            return Math.max(currMax, width);
+            const {cellClasses, rowClasses} = valueClassMap[value];
+
+            // 4b) Loop through all combinations of cell and row classes applied to this value,
+            // and return the largest.
+            let ret = 0;
+            rowClasses.forEach(rowClass => {
```

Comment:
this seems potentially expensive, and may even be testing combinations that don't exist, no?